independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/jaxrs/AbstractResponseBuilder.java

```diff
@@ -185,7 +185,7 @@ public AbstractResponseBuilder clone() {
         responseBuilder.status = status;
         responseBuilder.reasonPhrase = reasonPhrase;
         responseBuilder.entity = entity;
-        responseBuilder.metadata = new QuarkusMultivaluedHashMap<>();
+        responseBuilder.metadata = new CaseInsensitiveMap<>();
```

Comment:
How does this relate with the change?

Reply:
It's necessary in order to narrow the type of the map - it should have been done anyway

Reply:
Also see my comment below